pkg/mkimage-raw-efi/Dockerfile

```diff
@@ -23,16 +23,16 @@ RUN make -C /tmp/mkinitfs install
 
 WORKDIR /out
 RUN echo "mtools_skip_check=1" >> etc/mtools.conf
-RUN mkdir -p efifs parts root bits config persist opt/pillar opt/debug lib/modules run sys
-COPY make-raw install grub.cfg.in UsbInvocationScript.txt ./
+RUN mkdir -p efifs parts root bits config persist opt/pillar opt/debug opt/interactive-installer lib/modules run sys
+COPY make-raw install grub.cfg.in UsbInvocationScript.txt parse-interactive-installer-conf ./
 
 # Before changing something here please take a look into the
 # images/rootfs.yml.in onboot section: the installer should
 # precede the storage-init container.
 #
 # now lets create an edge container
-# FIXME: 002-installer? why not linuxkit build?
-WORKDIR /eco/media/root-rw/root/containers/onboot/002-installer
+# FIXME: 003-installer? why not linuxkit build?
+WORKDIR /eco/media/root-rw/root/containers/onboot/003-installer
```

Comment:
Oy, this is hideous. And not at all the fault of this PR. Why do we do this? I will open a separate issue to address this.

Reply:
> Oy, this is hideous. And not at all the fault of this PR. Why do we do this? I will open a separate issue to address this.

because of cross-container dependencies. And linuxkit doesn't provide a way to do it better. (I do not remeber exact dependency though)

Reply:
I opened #4056 to track it. I hope that Roman (referenced there) knows and we can come up with a cleaner solution.